src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  height: 100%;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+  font-family: $main-font-family;
+  color: $main-color;
+
+  &__photo {
+    width: 100%;
+    height: 134px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $minor-color;
+    margin: 0 0 16px;
+  }
+
+  &__feedback {
+    margin-bottom: 24px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    align-items: flex-end;
+  }
+
+  &-amount {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    line-height: 18px;
+
+    &__price {
+      font-size: 12px;
+      font-weight: 400;
+      color: $minor-color;
+    }
+
+    &__sum {
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    background-color: #00acdc;
+    text-decoration: none;
+    color: #fff;
+    height: 40px;
+    box-sizing: border-box;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    align-items: center;
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+
+    &:hover {
+      color: #00acdc;
+      background-color: #fff;
+    }
```

Comment:
On hover, the border color should also change to `#00acdc`. You can achieve this by adding `border-color: #00acdc;` inside the `&:hover` pseudoclass.